src/trade/tradeDialog.es6

```diff
@@ -508,15 +508,15 @@ function init_state(available,root, dialog, symbol, contracts_for_spot){
   };
 
   state.date_expiry.update = function (date_or_hour) {
-    var expiry = state.date_expiry;
+    let expiry = state.date_expiry;
```

Comment:
Do we need to reference all of the `state.date_expiry` object? Better to make a copy of the property that you need (to protect against unintended side effects)